src/component/app/app.component.jsx

```diff
@@ -2,21 +2,28 @@ import React from 'react'
 import { Provider } from 'react-redux'
 import { ThemeProvider } from 'styled-components'
 import theme from '../../config/theme'
+import * as S from './app.style'
```

Comment:
Why import * (all) as S if there is only one export ?

Reply:
cause will be more